File: Data Structures & Algorithms/products-of-array-discluding-self/submission-12.cs
public class Solution {
    public int[] ProductExceptSelf(int[] nums) {
        int result = 0;
        for (int left = 0; left < num.Length; left++) {
            for (int right = 0; right < num.length; right++) {
                result = result * nums[right];
            }
            res[left] = result
        }
        return res
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-12.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in len(nums):
            j = j+1 
            for j in len(nums):
                proposed = num[i] + num[j]
                if proposed == target:
                    return [i, j]
                
        return false